pysatCDAAC/instruments/cosmic_gps.py

```diff
@@ -20,66 +20,72 @@
     'gps' for Radio Occultation profiles
 tag
     Select profile type, or scintillation, one of:
-    {'ionprf', 'sonprf', 'wetprf', 'atmprf', 'scnlv1'}
-sat_id
+    ['ionprf', 'wetprf', 'atmprf', 'eraprf', 'gfsprf',
+    'ionphs', 'podtec', 'scnlv1']
```

Comment:
Recommend making this a list with descriptions:
```
'ionprof' - Ionospheric profile
...
```

Reply:
I updated the description but I switched to a dict and specifically referenced using one of the listed keys. I worried that a list of the keys plus description in a list could potentially throw off the newest of users.